### PR TITLE
texlive: Depends on perl for Linuxbrew

### PR DESCRIPTION
**Pull request by dholm**

### CODE REVIEW

**Comment by dholm** [comment]
https://gist.github.com/dholm/ffeef6884d6f0288d5b505a73998aeb4

File: Formula/texlive.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ class Texlive < Formula
   option "with-basic", "install plain and latex"
   option "with-minimal", "install plain only"
 
+  depends_on :perl => ["5.14", :build] if OS.linux?
```
Comment (sjackman) [inline Formula/texlive.rb]: Please change this to `unless OS.mac?`

Comment (sjackman) [inline Formula/texlive.rb]: Please change this to `unless OS.mac?`

Comment (dholm) [inline Formula/texlive.rb]: Done!

**Comment by sjackman** [comment]
Merged. Thanks again for your contribution to Linuxbrew, David! 👏
For future contributions, the preferred commit log message is:
```
texlive: Depends on perl for Linuxbrew
```

**Pull request merged**